script/test/fixtures/buildargs/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "2"
+
+services:
+    foo:
+        build: 
+          context: "./build"
+          args:
+            NAME: web
+        command: "sleep 3600"
```

Comment:
Would be good to test all combinations, ex:
```yaml
args:
 NAME: web
```

```yaml
args:
 - NAME=web
```
```
```yaml
args:
 - NAME
```
```yaml
args:
 NAME:
```